test/health_cards/covid_health_card_test.rb

```diff
@@ -8,20 +8,27 @@ class COVIDHealthCardTest < ActiveSupport::TestCase
     @card = rails_issuer.create_health_card(@bundle, type: HealthCards::COVIDHealthCard)
   end
 
+  class COVIDHealthCardSame < HealthCards::COVIDHealthCard; end
+
+  class COVIDHealthCardChanged < HealthCards::COVIDHealthCard
+    fhir_version '4.0.2'
+    additional_types 'https://smarthealth.cards#test'
+  end
+
   test 'is of custom type' do
     assert @card.is_a?(HealthCards::COVIDHealthCard)
   end
 
   test 'includes correct types' do
-    HealthCards::COVIDHealthCard.types.include?(HealthCards::CardTypes::VC_TYPE[0])
+    HealthCards::COVIDHealthCard.types.include?('https://smarthealth.cards#health-card')
     HealthCards::COVIDHealthCard.types.include?('https://smarthealth.cards#covid19')
   end
 
   test 'includes required credential attributes in hash' do
     hash = @card.to_hash
     type = hash.dig(:vc, :type)
     assert_not_nil type
-    assert_includes type, HealthCards::CardTypes::VC_TYPE[0]
+    assert_includes type, 'https://smarthealth.cards#health-card'
     assert_includes type, 'https://smarthealth.cards#covid19'
     assert_includes type, 'https://smarthealth.cards#immunization'
```

Comment:
Should we implement types so that `#immunization` and `#laboratory` only appear if those resources are present, or is @danlee1025 covering that in [DVCI-87](https://jira.mitre.org/browse/DVCI-87)?

Right now, a patient bundle with no lab nor immunization data still includes these types, which throws warnings when I test with the Microsoft Verifier portal.

Reply:
I think that's out of scope for this PR, as I'm only trying to address inheritance issues, but it would make sense for DVCI-87

Reply:
Actually, on second thought since theoretically this we would end up with invalid HC examples if I leave it as is, I might revisit this after I talk to @danlee1025.